tests/plugins/test_amp_plugins.py

```diff
@@ -83,3 +97,37 @@ def test_amp_gradient_unscale(tmpdir, accum: int):
         accumulate_grad_batches=accum,
     )
     trainer.fit(model)
+
+
+@RunIf(min_gpus=2, amp_apex=True, special=True)
+@pytest.mark.parametrize("amp_level", ['O2'])
+def test_amp_apex_ddp_fit(amp_level, tmpdir):
+
```

Comment:
We have another apex test in 
`tests/models/test_amp.py::test_amp_with_apex` It uses 1 gpu.
@tchaton your fix only applies to multi-gpu due to the dispatch, am I right?

Reply:
Yes.